src/js/models/menubar.js

```diff
@@ -260,19 +260,63 @@ define(function (require, exports, module) {
     };
 
     /**
+     * Replace keyChar and keyCode properties of menu shortcuts with localized
+     * strings.
+     *
+     * @private
+     * @param {Array.<object>} entries
+     * @param {Object.<string, string>} shortcuts
+     */
+    var _resolveShortcuts = function (entries, shortcuts) {
+        entries.forEach(function (entry) {
+            var id = entry.id;
+
+            if (entry.hasOwnProperty("shortcut")) {
+                var shortcut = entry.shortcut;
+                if (!shortcut.keyCode && !shortcut.keyChar) {
+                    throw new Error("Menu entry " + id + " has a shortcut without a key.");
+                }
+
+                if (!shortcuts) {
```

Comment:
Though I did push a fix to correct the JSDoc on that param to indicate that it is optional.